common/src/test/java/com/microsoft/identity/common/BrokerOAuth2TokenCacheTest.java

```diff
@@ -236,6 +243,7 @@ public void tearDown() throws Exception {
         }
 
         mApplicationMetadataCache.clear();
+        updateUseInMemoryCacheFlight(false);
```

Comment:
>updateUseInMemoryCacheFlight [](http://example.com/codeflow?start=8&length=28)

actually, I think you should call CommonFlightsManager.resetFlightsManager() after teach test.